src/core/include/openvino/op/squeeze.hpp

```diff
@@ -4,22 +4,34 @@
 
 #pragma once
 
+#include <functional>
+#include <utility>
+
 #include "openvino/op/op.hpp"
 
 namespace ov {
 namespace op {
 namespace v0 {
 /// \brief Squeeze operation.
 ///
+/// dimension. The solution mimics the PyTorch approach.
+///
 /// \ingroup ov_ops_cpp_api
 class OPENVINO_API Squeeze : public Op {
 public:
     OPENVINO_OP("Squeeze", "opset1");
 
+    /// \brief Constructs a squeeze operation.
     Squeeze();
-    Squeeze(const Output<Node>& data, const Output<Node>& axes);
+    /// \brief Constructs a squeeze operation.
+    ///
+    /// \param data Input tensor with data
+    /// \param axis The axis along which to squeeze the input tensor.
+    /// \param axis_skip_mode Shape inference result dynamic rank if selected axis has 1 in range of its dynamic
```

Comment:
I think the name of this parameter needs to match the name in spec

Reply:
Done